### PR TITLE
Create ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by linamnt**
Can you guys take a quick look to see if it's appropriate and what we're looking for?

### CODE REVIEW

**Comment by linamnt** [comment]
By guys, I mean @mbonsma and @QuLogic sorry!

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+`Please follow this template if you're posting an event, volunteer, job or other
+opportunity to the UofT Coders Study Group. 
+Not following this template may result in your issue being closed. 
+You can remove this header portion when submitting your issue.`
+
+**Your Name, or Organization Name**:
+
+**Contact info**:
+
+**Type (Event, Volunteer or Job Opportunity, or other (please specify))**:
```
Comment (mbonsma) [inline ISSUE_TEMPLATE.md]: Type of post (Event ...)

**Review by mbonsma — APPROVED** [review]
I like it! Maybe they don't need to keep the topic headings though, you could say something like "Make sure your post addresses the following points" or something like that.

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+```
```
Comment (QuLogic) [inline ISSUE_TEMPLATE.md]: Instead of making a code block, maybe wrap this in `<!--` and `-->` which will comment it out when the issue is created.

**Review by QuLogic — APPROVED** [review]

**Pull request merged**